t/t4215-log-bloom.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/sh
+
+test_description='git log for a path with bloom filters'
+. ./test-lib.sh
+
+GIT_TEST_COMMIT_GRAPH_BLOOM_FILTERS=0
+
+test_expect_success 'setup the full repo' '
+	mkdir full &&
+	cd "$TRASH_DIRECTORY/full" &&
```

Comment:
Don't use the "full" repo thing here, just use the repo the test lib already built for you. (This bad pattern is my fault in `t5318-commit-graph.sh` but have since learned better.)